internal/scylla/endpoints.go

```diff
@@ -65,6 +65,8 @@ func (c *Client) Connect(clusterID int64) (*model.ClusterConnection, error) {
 		return nil, err
 	}
 
+	fix_sf3112(&result) // TODO(rjeczalik): remove when #3112 at sf is fixed
```

Comment:
would be good to mention in which repo #3112 is

Reply:
done